source/includes/_documentation.md

```diff
@@ -4853,7 +4853,118 @@ The angular Button:
 **is()** | Assert action | TextAssert 
 
 #### <a href="https://github.com/jdi-testing/jdi-light/blob/master/jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/common/BasicButtonTests.java" target="_blank">Basic Button java tests examples</a>
----
+
+### Snack-bar
+
+#### <a href="https://material.angular.io/components/snack-bar/overview" target="_blank">Snack-bar overview</a>
+
+Snack-bar is located in the following class:
+
+   - __Java__: _com.epam.jdi.light.angular.elements.common.Snackbar_
+
+
+There are two different snack-bars in Angular: Basic and Snack-bar with a custom component:
+
+<br>
+
+![Basic snack-bar](../images/angular/basic_snackbar.png)
+
+```java 
```

Comment:
![image](https://user-images.githubusercontent.com/65406596/87923041-8a6bc080-ca85-11ea-8991-460ac7afb995.png)
I think, it would be better if, you raised the java example above.

Reply:
Will do
But this item in requirements is somewhat misleading:
![image](https://user-images.githubusercontent.com/32023395/87970624-aa72a280-cacc-11ea-9928-c912559bd977.png)